RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,94 @@ class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
-            string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            int firstPlayer = 0;
+            int secondPlayer = 0;
+            bool keepPlaying = true;
+            bool playAgain = true;
+            while (keepPlaying == true)
+            {
+                while (playAgain == true)
+                {
+                    Console.Clear();
+                    Console.WriteLine("Enter hand 1:");
+                    string hand1 = Console.ReadLine().ToLower();
+                    Console.WriteLine("Enter hand 2 or c for a computer player");
+                    string hand2 = Console.ReadLine().ToLower();
+                    string showWinner = (CompareHands(hand1, hand2));
+                    string[] splitResults = showWinner.Split('*');
+                    string winnerText = splitResults[0];
+                    int winnerAdd = Convert.ToInt32(splitResults[1]);
+                    Console.WriteLine(winnerText);
+                    if (winnerAdd == 1)
+                    {
+                        firstPlayer++;
+                    }
+                    else if (winnerAdd == 2)
+                    {
+                        secondPlayer++;
+                    }
+                    Console.WriteLine("The score is {0} to {1}", firstPlayer, secondPlayer);
 
+                    playAgain = false;
+                }
+                Console.WriteLine("Would you like to play again y/n");
+                bool playingagain = (restartGame(Console.ReadLine()));
+                if (playingagain)
+                {
+                    playAgain = true;
+                }
+            }
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
         }
-        
+        private static readonly Random getrandom = new Random();
         public static string CompareHands(string hand1, string hand2)
         {
             // Your code here
-            return hand1 + ' ' + hand2;
+            string hand2c = hand2;
+            string ifComputer = "Player 2";
+            if (hand2 == "c")
+            {
+                int computer = getrandom.Next(1, 3);
```

Comment:
Read up on this method, it does not do what you think it does the way it does it

![does not mean](https://bigmedium.com/bm.pix/1r2ate.jpg)